### ISSUE TITLE
LCC wrfout_d03 LANDMASK doesn't match background map

### ISSUE THREAD

**Issue by lizadams** [CEMPD/VERDI#139]
![toolsuivsverdi_wrfout_d03_landmask](https://cloud.githubusercontent.com/assets/1183863/19544718/e1f8b732-964e-11e6-9d2c-81089f8400ed.png)
![screen shot 2016-10-19 at 10 51 43 pm](https://cloud.githubusercontent.com/assets/1183863/19544719/e3bd2b5c-964e-11e6-85df-bbb18174570f.png)
![screen shot 2016-10-20 at 12 11 30 am](https://cloud.githubusercontent.com/assets/1183863/19546016/d09180e0-9659-11e6-9e5c-9b5542819ee9.png)

**Comment by lizadams** [comment]
Well, I may have figured out the source of the problem using a python script provided to me by Barron Henderson.

In the calculation, of gx0, gy0 , the gy0 was set to the MOAD_CEN_LAT
When I changed it to using the CEN_LAT instead, then the gridded data
matched up with the basemap
CEN_LAT = 39.13765f ;
MOAD_CEN_LAT = 40.f ;

corrected:  wrf_basemap.geo.py
gx0, gy0 = proj(wrffile.CEN_LON, wrffile.CEN_LAT)

original:   wrf_basemap.py
gx0, gy0 = proj(wrffile.CEN_LON, wrffile.MOAD_CEN_LAT)

We are still using MOAD_CEN_LAT in the projection string, and I don't know
if that is correct or not.
For the calculation of the grid, it definitely was making a difference.
I am attaching the new images, as they now look consistent between the
contour filled and the tile plots generated by Barron's python script.

I will check in the formulas in VERDI to see what is being used. - Note, I have an issue with editing the WRFConvention.java file as it is currently in the netcdfAll-4.5.5-SNAPSHOT.jar
When I tried editing this file and then rebuilding the jar file on Killdevil and using it, it worked for the WRF files but caused an error when creating the plots for the MPAS files.  I asked Catherine for help in how to use the local WRFConvention.java file so that we do not need to edit the jar.
